tests/test_main.py

```diff
@@ -50,6 +50,27 @@ def test_main_without_dummy_data_file(tmp_path):
     assert output.read_text("utf-8") == 'patient_id\n""\n'
 
 
+def test_main_with_dummy_data_file_to_stdout(capsys, tmp_path):
+    input_ = tmp_path / "query.sql"
+    input_.write_text(
+        f"-- {T1OOS_TABLE} intentionally not excluded\nSELECT Patient_ID FROM Patient",
+        "utf-8",
+    )
+    csv_lines = "patient_id\n1\n"
+    dummy_data_file = tmp_path / "dummy_data.csv"
+    dummy_data_file.write_text(csv_lines, "utf-8")
+    main.main(
+        {
+            "dsn": None,
+            "input": input_,
+            "dummy_data_file": dummy_data_file,
+            "output": None,
+        }
+    )
+    stdout = capsys.readouterr().out
+    assert stdout.replace("\r", "") == csv_lines
```

Comment:
Okay, so `sys.stdout`'s newline is `\r\n` and it's hard to change. My preference would be

```suggestion
    assert stdout == ""patient_id\r\n1\r\n""
```

to remove a small amount of logic from this test.

Reply:
Is that the same across all platforms?

Reply:
I spent a bit too long digging into this, but still can't say for sure. If not, then it would be good to flush out. Not essential, though.

Reply:
Sorry, I jumped the gun on merging this!